packages/adapter-static/package.json

```diff
@@ -3,11 +3,21 @@
 	"version": "1.0.0-next.4",
 	"scripts": {
 		"lint": "eslint --ignore-path .gitignore \"**/*.{ts,js,svelte}\" && npm run check-format",
+		"check": "tsc",
 		"format": "prettier --write . --config ../../.prettierrc --ignore-path .gitignore",
-		"check-format": "prettier --check . --config ../../.prettierrc --ignore-path .gitignore"
+		"check-format": "prettier --check . --config ../../.prettierrc --ignore-path .gitignore",
+		"test": "uvu test test.js"
 	},
 	"devDependencies": {
 		"@sveltejs/kit": "workspace:*",
-		"typescript": "^4.2.3"
+		"playwright-chromium": "^1.10.0",
+		"port-authority": "^1.1.2",
+		"sirv": "^1.0.11",
+		"typescript": "^4.2.4"
+	},
+	"type": "module",
+	"main": "index.cjs",
+	"exports": {
+		"require": "./index.cjs"
```

Comment:
Is this the shape this needs to take? It doesn't need to be `exports: { '.': { require: './index.cjs' } }`?

Reply:
`.` is assumed at the top level